docs/stats.md

```diff
@@ -80,8 +83,8 @@ whereas {ref}`sec_stats_sample_sets_multi_way` compare 2 or more sets of samples
 * Multi site
     * {meth}`~LdCalculator` (note this is soon to be deprecated)
 
-:::{todo}
-Add information about the new IBD stats.
+:::{seealso}
+The identity by descent methods **link**.
```

Comment:
Can link to "{ref}`sec_ibd`" now can't we?

Reply:
I've taken it out, as on consideration I don't think it's really "stats", but we can refer to it later I guess.